tests/test_backward_compat.py

```diff
@@ -0,0 +1,25 @@
+import torch
+
+from netam.framework import load_crepe
+from netam.sequences import set_wt_to_nan
+
+
+# The outputs used for this test are produced by running
```

Comment:
This is great but perhaps give a little more context for when I forget what old means?